### PR TITLE
minor: make Parquet CLI input args consistent

### LINKED ISSUE #3785: Make Parquet CLI args consistent
**Is your feature request related to a problem or challenge? Please describe what you are trying to do.**

The  current input argument format is inconsistent across CLI tools. E.g., parquet-layout does not need passing "-f" or "--file" for the input file name. This should be consistent across the binary tools.

**Describe the solution you'd like**

One solution is to flag all clap arguments with short and long.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by XinyuZeng**
# Which issue does this PR close?

Closes #3785.

# Rationale for this change
 

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

**Comment by tustvold** [comment]
What do you think of unifying in the opposite direction, i.e. making the other tools take positional arguments?

**Comment by XinyuZeng** [comment]
Positional arguments may not be well suited for tools like parquet-fromcsv and parquet-rewrite, which have too many arguments (and many file names). But for the tools that only require one file to read, I think getting rid of "-f" makes it handier. For those tools, we can make filename positional and leave other options there.

e.g.
```
parquet-read [OPTIONS] <FILE_NAME>
```

**Comment by tustvold** [comment]
I think it is fine for the more complex tools to have a different argument pattern from those that take a single input file

File: parquet/src/bin/parquet-index.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ struct Args {
     #[clap(help("Path to a parquet file"))]
     file: String,
 
-    #[clap(help("Column name to print"))]
```
Comment (tustvold) [inline parquet/src/bin/parquet-index.rs:51]: Why this change?

Comment (XinyuZeng) [reply parquet/src/bin/parquet-index.rs]: Fixed. Also made args in bloom filter CLI positional because they are not optional

**Review by tustvold — APPROVED** [review]

**Comment by tustvold** [comment]
Thank you :+1:
